dude.py

```diff
@@ -337,22 +429,14 @@ def ldap_search(users: list, unknown: list) -> list:
 
     # Remove all users without query results
     users = [user for user in users if user[1]]
-    if len(users) < len(users_old):
-        non_found = [
-            old_user for old_user in users_old if old_user not in
-            [user[0] for user in users]
-        ]
-        if len(non_found) >= 1:
-            print(f"The following user could not be found in the LDAP: \"{non_found[0]}\"")
-            raise RuntimeError(
-               f"The following user could not be found in the LDAP: \"{non_found[0]}\""
-            )
-        userstring = "\"" + "\", \"".join(non_found) + "\""
-        raise RuntimeError(
-            f"The following user could not be found in the LDAP: {userstring}"
-        )
-    print("LDAP: ")
-    print(users)
```

Comment:
Just because you only need one user at a time doesnt mean that this functionality is unneeded, please restore the code for multiple users